tools/ot/src/main/java/dk/alexandra/fresco/tools/ot/base/Ot.java

```diff
@@ -0,0 +1,29 @@
+package dk.alexandra.fresco.tools.ot.base;
+
+import dk.alexandra.fresco.framework.util.StrictBitVector;
+
+/**
+ * Oblivious Transfer interface.
+ */
+public interface Ot {
+
+  /**
+   * Send two possible messages for the recipient to choose from.
+   *
+   * @param messageZero
+   *          Message zero to send
+   * @param messageOne
+   *          Message one to send
+   */
+  void send(StrictBitVector messageZero, StrictBitVector messageOne);
```

Comment:
I think we can go with the interface here, I.e. StrictBitVector

Reply:
Typo, BitVector

Reply:
I am not sure it is a good idea. The reason being that the OT extension was designed with the assumption that vectors contains an 8-product of bits. Thus to use the interface we either have to change the code we already have to handle the case where the input is not an 8 product or throw an exception in this case.